Bio/Align/__init__.py

```diff
@@ -2341,7 +2342,12 @@ def _format_pretty(self, matrix=None):
                 raise ValueError("Inconsistent coordinates")
         prefix_width = 10
         position_width = 10
-        line_width = 80
+
+        if not hasattr(self, "terminal_columns") or self.terminal_columns is None:
```

Comment:
It is better to have a class variable `terminal_columns` on the `Alignment` class.